### DISCUSSION TITLE
Tabs KeepAlive

### DISCUSSION THREAD

**Discussion by psychonetic** [tailwindlabs/headlessui#743] [Ideas]
At the moment only the active tab is rendered. It would be awesome to support also keep alive for tabs, so instead of rendering only the active one all tabs are rendered and none active tabs will be hidden by setting `display: hidden`.

Use case: e.g. form using multiple tabs, which is quite common.

**Comment by RobinMalfait** [comment]
Hey!

Yep, this is already doable with the `unmount={false}` prop!
See: https://github.com/tailwindlabs/headlessui/discussions/740#discussioncomment-1195341

**Reply by Denispo** [reply]
`unmount` property is not mentioned in documentation

**Comment by mitar** [comment]
I made also #2111.
